LICENSE

```diff
@@ -1,7 +1,7 @@
                     GNU AFFERO GENERAL PUBLIC LICENSE
                        Version 3, 19 November 2007
 
- Copyright (C) 2007 Free Software Foundation, Inc. <http://fsf.org/>
+ Copyright (c) 2007 Free Software Foundation, Inc. <http://fsf.org/>
```

Comment:
These lines are truly the highlight of the PR.

Reply:
Agreed, it was driving me to distraction.

Reply:
Finally I can sleep again!